german/es-wird-kalt/Kälte

```diff
@@ -0,0 +1,3 @@
+Die Beine werden langsam kalt. Sie wissen, dass Sie sich bewegen müssen.
```

Comment:
I'm not sure why Casey said this directory already contained a "page" of the story - that looks like a bug. The other two comments are valid, though.